feature-group-definitions/css-nesting.yml

```diff
@@ -0,0 +1,2 @@
+spec: https://drafts.csswg.org/css-nesting-1/
```

Comment:
`nested-rules.yml` or `nested-styles.yml`. More fodder for the the naming convention discussion, but I think this PR is having me think of features as nouns and gerunds as cheating. 😄

Reply:
One point in favor of "nesting" is that the feature is known as such in [Sass](https://sass-lang.com/guide), and is also called this in [State of CSS](https://2022.stateofcss.com/en-US/usage/).

Reply:
Oh cool—let's go with `nesting` then. I think names in common use ought to outweigh formalisms (even if I really, really like formalisms).